Makefile

```diff
@@ -33,8 +33,9 @@ clean: ## Delete intermediate build artifacts
 .PHONY: generate
 generate: $(BIN)/buf $(BIN)/license-header ## Regenerate code and license headers
 	rm -rf gen
-	buf generate buf.build/bufbuild/protovalidate:$(PROTOVALIDATE_VERSION)
-	buf generate buf.build/bufbuild/protovalidate-testing:$(PROTOVALIDATE_VERSION)
+	$(BIN)/buf generate buf.build/bufbuild/protovalidate:$(PROTOVALIDATE_VERSION)
+	$(BIN)/buf generate buf.build/bufbuild/protovalidate-testing:$(PROTOVALIDATE_VERSION)
+	$(BIN)/buf generate
```

Comment:
I agree that it's helpful to have local proto files for debugging. But `PROTOVALIDATE_VERSION` _and_ `buf.lock` competing over the exact version seems like a footgun. 

Maybe the least controversial thing to do is to cut a protovalidate release that contains the new test protos?

Reply:
I’m not following. pv-go is doing this exact thing with no issues. I would really prefer for this to follow that pattern. Being able to change protos while debugging issues is much quicker than constantly having to cut protovalidate releases.

Reply:
It's good that we haven't had an issue with it so far. I didn't realize the Go implementation already does it. LGTM, and hopefully find a better solution down the road.